test/benchee/formatters/console_test.exs

```diff
@@ -21,6 +21,31 @@ defmodule Benchee.Formatters.ConsoleTest do
     assert Regex.match?(~r/Third/,  result_3)
   end
 
+  test "adjusts the label width to longest name" do
+    third_name = String.duplicate("a", 40)
+    first  = {"First",  %{average: 100.0, ips: 10_000.0,
+                          std_dev_ratio: 0.1, median: 90.0}}
+    second = {"Second", %{average: 200.0, ips: 5_000.0,
+                          std_dev_ratio: 0.1, median: 195.5}}
+    third  = {third_name,  %{average: 400.0, ips: 2_500.0,
+                             std_dev_ratio: 0.1, median: 375.0}}
+
+    # Just normally long names, expect default width of 30 + 13
```

Comment:
I didn't find a way to grap the console formatter attribute outside of that module, except to define a new function. So I skipped it, but added it to the assert function instead, which already has to adjust for the ips column.